backend/src/main/scala/cromwell/backend/BackendCacheHitCopyingActor.scala

```diff
@@ -7,7 +7,9 @@ import cromwell.core.simpleton.WomValueSimpleton
 object BackendCacheHitCopyingActor {
   final case class CopyOutputsCommand(womValueSimpletons: Seq[WomValueSimpleton], jobDetritusFiles: Map[String, String], returnCode: Option[Int])
 
-  final case class CopyingOutputsFailedResponse(jobKey: JobKey, cacheCopyAttempt: Int, failure: CacheCopyError)
+  // 'declined' means that the copying actor declined to attempt copying the cache hit. Currently this is true only
+  // for blacklisting reasons.
+  final case class CopyingOutputsFailedResponse(jobKey: JobKey, cacheCopyAttempt: Int, failure: CacheCopyError, declined: Boolean = false)
```

Comment:
I didn't like having a new term for this but I wasn't sure if "blacklisted" was sufficiently clear. I wanted something to make it more clear that the the copying actor was saying "I did not try to copy this hit because either the hit or the bucket that contained it was blacklisted" versus "I tried to copy this hit but that didn't work out so I blacklisted the hit and possibly the bucket".

Reply:
Maybe `deliberatelyNotTried`? My first reading of `declined` would be that we got some kind of network error connecting to GCS